.github/workflows/release.yml

```
@@ -0,0 +1,125 @@
+name: Create Zeek release
+
+on:
+  workflow_dispatch:
+    inputs:
+      zeek_ref:
+        required: false
+        type: string
+        default: 'master'
+      build_zeek_ref:
+        required: false
+        type: string
+        default: 'main'
+      release_tag:
+        required: true
+        type: string
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    strategy:
+      matrix:
+        platform: [macos-12, ubuntu-20.04, windows-2019]
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - name: Checkout build-zeek
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ inputs.build_zeek_ref }}
+
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version-file: go.mod
```

This isn't really a Go module so there's no need for go.mod.
```suggestion
        go-version: 1.21
```